Week4/GamesList/app/src/main/java/com/raywenderlich/gameslist/GameDetailsViewHolder.kt

```diff
@@ -0,0 +1,16 @@
+package com.raywenderlich.gameslist
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class GameDetailsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    var gameCover = itemView.findViewById<ImageView>(R.id.gameCover)

```

Comment:
Synthetics!

Reply:
Also, you can put your display logic into the Holder, and that'll make it much easier to separate the code!